boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -276,14 +276,30 @@ You should see the following message on the console:
 Building a secure/non-secure with Arm |reg| TrustZone |reg|
 -----------------------------------------------------------
 
-The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q,
-using the ``nucleo_l552ze_q_ns`` target. When building a ``*_ns`` image with TF-M,
-a ``build/tfm/install/postbuild.sh`` bash script will be run as a post-build step
-to make some required flash layout changes. The ``build/tfm/regression.sh`` script
-will need to be run to perform device initialization, and then run ``west flash --hex-file build/tfm_merged.hex``
-to flash the board.
-
-Check the ``build/tfm/`` directory to ensure that the commands required by these scripts
+The TF-M integration sample :ref:`tfm_ipc` can be run on a ST Nucleo L552ZE Q.
+In TF-M configuration, Zephyr is run on the non-secure domain. A non-secure image
+could be generated using ``nucleo_l552ze_q_ns`` as build target.
+
+.. code-block:: bash
+
+   $ west build -b nucleo_l552ze_q_ns path/to/source/directory
+
+Note: When building the ``*_ns`` image with TF-M, ``build/tfm/postbuild.sh`` bash script
+is run automatically in a post-build step to make some required flash layout changes.
+
+Once the build is completed, run the following script to perform option bytes initialization.
```

Comment:
I'm not sure of the exact sense here, but if we're changing 'option bytes' this might be clearer:

```suggestion
Once the build is completed, run the following script to initialize the option bytes.
```